crates/exec-wasmtime/src/config.rs

```diff
@@ -188,7 +195,8 @@ mod test {
                 File::Listen {
                     name: "X".into(),
                     port: 9000,
-                    prot: Protocol::Tcp
+                    prot: Protocol::Tcp,
+                    addr: "::".to_string()
```

Comment:
Shouldn't this be using the helper function if we keep it?

Reply:
why?  ... changed it to `"::1"`

Reply:
yes